template-compose/app/src/main/AndroidManifest.xml

```diff
@@ -1,20 +1,25 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="co.nimblehq.template.compose">
+    xmlns:tools="http://schemas.android.com/tools">
 
     <uses-permission android:name="android.permission.INTERNET" />
 
     <application
         android:name=".MainApplication"
         android:allowBackup="true"
+        android:dataExtractionRules="@xml/data_extraction_rules"
+        android:fullBackupContent="@xml/backup_rules"
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
         android:networkSecurityConfig="@xml/network_security_config"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
-        android:theme="@style/AppTheme">
+        android:theme="@style/AppTheme"
+        tools:targetApi="31">
```

Comment:
May I know the purpose of this `tools:targetApi="31"`? 🙏🏻

Reply:
@lydiasama To bypass this warning, it explains the `dataExtractionRules` attr is used on API 31 only and will be ignored on lower APIs. The `tools:targetApi="31"` is default generated for the new project by AS with Gradle 8.

<img width="789" alt="image" src="https://github.com/nimblehq/android-templates/assets/16315358/8e2924a5-5323-4196-b5e2-a288b72740d9">